tests/test_sqlalchemy_athena.py

```diff
@@ -167,6 +159,15 @@ def test_get_table_names(self, engine, conn):
             insp.get_table_names(schema=SCHEMA),
```

Comment:
With respect to the issue reported, we should also check that no view is listed:
```python
         table_names = insp.get_table_names(schema=SCHEMA)
         self.assertIn("many_rows", table_names)
         self.assertNotIn("view_one_row", table_names)
```